docs/CONTRIBUTING.md

```diff
@@ -31,13 +31,23 @@ from onnx root dir should work.
 
 ## Generated operator documentation
 
-[Operator docs in Operators.md](Operators.md) are automatically generated based on C++ operator definitions and backend Python snippets. To refresh these docs, run the following commands from the repo root and commit the results. Note `ONNX_ML=0` updates Operators.md whereas `ONNX_ML=1` updates Operators-ml.md:
+[Operator docs in Operators.md](Operators.md) are automatically generated based on C++ operator definitions and backend Python snippets. To refresh these docs, run the following commands from the repo root and commit the results. 
```

Comment:
CONTRIBUTING.md should not change in this PR.

Reply:
I reverted the changes.